server/raft_transport.go

```diff
@@ -159,11 +159,13 @@ func (t *rpcTransport) processQueue(nodeID roachpb.NodeID, storeID roachpb.Store
 	done := make(chan *gorpc.Call, cap(ch))
 	var req *storage.RaftMessageRequest
 	protoResp := &storage.RaftMessageResponse{}
+	raftIdleTimer := time.NewTimer(raftIdleTimeout)
 	for {
+		raftIdleTimer.Reset(raftIdleTimeout)
```

Comment:
I'm not sure if this is functionally equivalent. The old code would restart the timer on every iteration. But it looks like this new pattern introduces a small race where the timer can fire just as a request is received. The timer internally sends to `raftIdleTimer.C` and on the next iteration we call `Reset`, but that doesn't clear the channel. I'm not familiar enough with this code to know if that race is innocuous.